### PR TITLE
Create quickstartlaunch.md

### PR DESCRIPTION
**Pull request by lizygeogy**

### CODE REVIEW

**Comment by lizygeogy** [comment]
@DT-Priyanka

File: docs/quickstartlaunch.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# Quick Start Guide for Launching DataTorrent RTS Applications
```
Comment (DT-Priyanka) [inline docs/quickstartlaunch.md]: We should have two independent docs, one for ATO and other for FPA. Also as @vijaydt  suggested please add links to these docs in our applications guide for ATO & FPA

File: docs/quickstartlaunch.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+   - Click **configure**.
+   - From the **Select Replacement Applications** drop down, select the corresponding configuration name for both the dashboards.
+   - Click **Save**.<configurepackagedashboardfpa>
+1. Save the configuration.
+   - Click **Save.**
+   - Click **launch** to launch the application.<saveandlaunchfpa>
+
+## Fraud Prevention Application's Data Generator
+
+1. Create **New Configuration** for the OmniChannelFraudPreventationDataGenerator.
```
Comment (DT-Priyanka) [inline docs/quickstartlaunch.md]: for the OmniChannelFraudPreventionDataGenerator application.

**Comment by DT-Priyanka** [comment]
I assumed we have diagrams.

**Pull request merged**